chaosazure/machine/actions.py

```diff
@@ -117,6 +118,58 @@ def start_machine(filter: str = None,
     __start_stopped_machines(client, stopped_machines)
 
 
+def stress_cpu(filter: str = None,
+               duration: int = 120,
+               configuration: Configuration = None,
+               secrets: Secrets = None):
+    """
+    Stress CPU up to 100% at random machine.
+
+    Parameters
+    ----------
+    filter : str
+        Filter the virtual machines. If the filter is omitted all machines in
+        the subscription will be selected as potential chaos candidates.
+        Filtering example:
+        'where resourceGroup=="myresourcegroup" and name="myresourcename"'
+    """
+    logger.info(
```

Comment:
We tend to keep ` info`  for Chaos Toolkit messages and use ` debug`  for extensions inner messages because when running we want to see the fow of the experiment rather than how it gets there :)

do you have any thoughts on this one? :)

Reply:
make sense :) will update log level